src/main/resources/swagger.yaml

```diff
@@ -0,0 +1,949 @@
+              - redirect_uris
+              type: object
+              properties:
+                redirect_uris:
+                  type: array
+                  description: Request parameters as JSON object with data described by Connect Client Registration Specification..
+                  example:
+                  - https://client.example.org/cb
+                  items:
+                    type: string
+      parameters:
+        - name: client_id
+          in: query
+          required: true
+          description: Client ID that identifies client that must be updated by this request.   
+          schema:
+            type : string  
+        - name: Authorization
+          in: header
+          required: true
+          description: Access Token that is used at the Client Configuration Endpoint   
+          schema:
+            type : string      
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: ClientResponse
+                $ref: '#/components/schemas/ClientResponse'                                        
+        400:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        302:
+          description: Access Denies.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse' 
+  /register(get):
+    get:
+      tags:
+      - developers
+      summary: The Client Registration Endpoint to retrieve the Client Metadata for a previously registered client.
+      description: Reads client info.
+      operationId: requestClientRead
+      parameters:
+        - name: client_id
+          in: query
+          required: true
+          description: Client ID that identifies client.  
+          schema:
+            type : string      
+        - name: Authorization
+          in: header
+          required: true
+          description: Access Token that is used at the Client Configuration Endpoint   
+          schema:
+            type : string      
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                title: ClientResponse
+                $ref: '#/components/schemas/ClientResponse'                                        
+        401:
+          description: Invalid parameters are provided to endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'        
+        403:
+          description: The client is not allowed to delete itself.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse' 
+  /register(delete):
```

Comment:
no such endpoint

Reply:
Rectified, have now kept only one endpoint /register and listed the available operations